foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/platform/database/Oracle8Platform.java

```diff
@@ -14,12 +14,341 @@
+        return usesLocatorForLOBWrite;
+    }
+
+    /**
+     * PUBLIC: Return the BLOB/CLOB value limits on thin driver. The default
+     * value is 0. If usesLocatorForLOBWrite is true, locator will be used in
+     * case the lob's size is larger than lobValueLimit.
+     */
+    public int getLobValueLimits() {
+        return lobValueLimits;
+    }
+
+    /**
+     * PUBLIC: Set the BLOB/CLOB value limits on thin driver. The default value
+     * is 0. If usesLocatorForLOBWrite is true, locator will be used in case the
+     * lob's size is larger than lobValueLimit.
+     */
+    public void setLobValueLimits(int lobValueLimits) {
+        this.lobValueLimits = lobValueLimits;
+    }
+
+    /**
+     * INTERNAL: Used by Oracle platforms during reading of ResultSet to free
+     * temporary LOBs.
+     */
+    public void freeTemporaryObject(Object value) throws SQLException {
+        if (value == null) {
+            return;
+        }
+
+        final Class valClass = value.getClass();
+        final String valClassName = valClass.getName();
+        if ("oracle.sql.CLOB".equals(valClassName)) {
+            final Class clobClass = resolveClass("oracle.sql.CLOB");
+            if (clobClass == null) {
+                return;
+            }
+            try {
+                final Method isTemporaryMethod = clobClass.getMethod("isTemporary");
+                isTemporaryMethod.invoke(value);
+            } catch (Exception e) {
+                // TODO: Emit Warning
+                return;
+            }
+
+        } else if ("oracle.sql.BLOB".equals(valClassName)) {
+            final Class blobClass = resolveClass("oracle.sql.BLOB");
+            if (blobClass == null) {
+                return;
+            }
+            try {
+                final Method isTemporaryMethod = blobClass.getMethod("isTemporary");
+                isTemporaryMethod.invoke(value);
+            } catch (Exception e) {
+                // TODO: Emit Warning
+                return;
+            }
+        }
+    }
+
+    private Class resolveClass(String className) {
+        Class cls = null;
+
+        ClassLoader tcl = null;
+        if (PrivilegedAccessHelper.shouldUsePrivilegedAccess()) {
+            try {
+                tcl = (ClassLoader) AccessController
+                        .doPrivileged(new PrivilegedGetContextClassLoader(Thread.currentThread()));
+            } catch (PrivilegedActionException exception) {
+                // should not be thrown
+            }
+        } else {
+            tcl = PrivilegedAccessHelper.getContextClassLoader(Thread.currentThread());
+        }
+
+        try {
+            cls = tcl.loadClass(className);
+        } catch (ClassNotFoundException e) {
+            // Fall back to regular ClassLoader
+            try {
+                cls = Oracle8Platform.class.getClassLoader().loadClass(className);
+            } catch (ClassNotFoundException e1) {
+                // TODO: Log a warning
```

Comment:
Maybe this? : 
``` java
AbstractSessionLog.getLog().logThrowable(SessionLog.WARNING, null, e);
```

Reply:
Got rid of it, don't need the reflection code anymore.